File: Exercises/JavaScript/JavaScript_Basics/Arrays/ex8.js
// How can you check whether a variable holds a value that is an array? For example, imagine you start writing a function and want to check whether its argument is an array:

function filter(input) {
  // Is input an array?
  console.log(Array.isArray(input));
}

filter([1,2,3]);
filter('1,2,3');

File: Exercises/Python/Python_Basics/8_Lists/ex8.py
'''
How can you check if a variable holds a value that is a list? Given two variables, verify whether the values they hold are lists.
'''

some_value1 = [0, 1, 0, 0, 1]
some_value2 = 'I leave you my Kingdom, take good care of it.'

def is_list(var):
    return isinstance(var, list)

print(is_list(some_value1))
print(is_list(some_value2))
